Documentation/contributing/release/backports.rst

```diff
@@ -184,14 +184,30 @@ the labels for the PRs that are backported, based on the
 
       $ GITHUB_TOKEN=xxx contrib/backporting/submit-backport
 
+The script takes up to four positional arguments:
+
+   .. code-block:: bash
+
+      usage: submit-backport [branch version] [pr-summary] [upstream remote] [your remote]
+
+- The first parameter is the version of the branch against which the PR should
+  be done, and defaults to the version passed to ``start-backport``.
+- The second one is the name of the file containing the text summary to use for
+  the PR, and defaults to the file created by ``start-backport``.
+- The third one is the name of the git remote of the upstream base branch.
+  It defaults to the git remote of ``github.com/cilium/cilium``.
```

Comment:
Do you have a use case in mind for changing from the default, here? Aren't we always pushing the backport PR to `cilium/cilium`?